Tone/instrument/CymbalSynth.js

```diff
@@ -0,0 +1,304 @@
+	 *                             see defaults below
+	 *  @example
+	 * var cymbal = new Tone.CymbalSynth().toMaster();
+	 * cymbal.harmonicity.value = 10;
+	 * cymbal.triggerAttack();
+	 */
+	Tone.CymbalSynth = function(options){
+
+		options = this.defaultArg(options, Tone.CymbalSynth.defaults);
+		Tone.Instrument.call(this, options);
+
+		/**
+		 * The initial strike component of the cymbal hit.
+		 * @type {CymbalComponent}
+		 */
+		this.strike = new CymbalComponent({
+			"volume" : options.strike.volume,
+			"cutoff" : options.strike.cutoff,
+			"frequencyScalar" : 1.886,
+			"resonance" : options.strike.resonance,
+			"resonanceScalar" : 0.533,
+			"envelope" : {
+				"attack": options.strike.envelope.attack, 
+				"decay": options.strike.envelope.decay, 
+			}
+		}).connect(this.output);
+
+		/**
+		 * The body portion of the cymbal hit.
+		 * @type {CymbalComponent}
+		 */
+		this.body = new CymbalComponent({
+			"volume" : options.body.volume,
+			"cutoff" : options.body.cutoff,
+			"frequencyScalar" : 1.643,
+			"resonance" : options.body.resonance,
+			"resonanceScalar" : 0.333,
+			"envelope" : {
+				"attack": options.body.envelope.attack, 
+				"decay": options.body.envelope.decay, 
+			}
+		}).connect(this.output);
+
+		/**
+		 *  [harmonicity description]
+		 *  @type {Positive}
+		 *  @signal
+		 */
+		this.harmonicity = new Tone.Signal(options.harmonicity);
+		this.harmonicity.units = Tone.Type.Positive;
+
+		/**
+		 *  The base frequency for the series of oscillators. 
+		 *  All oscillators scale in a harmonic or inharmonic pattern
+		 *  depending on the value of the harmonicity.
+		 *  @type {Frequency}
+		 *  @signal
+		 */
+		this.frequency = new Tone.Signal(options.frequency, Tone.Type.Frequency);
+
+		/**
+		 *  Inharmonic ratio of frequencies based on the Roland TR-808
+		 *  Taken from https://ccrma.stanford.edu/papers/tr-808-cymbal-physically-informed-circuit-bendable-digital-model
+		 *  @private
+		 *  @static
+		 *  @type {Array}
+		 */
+		this.inharmRatios = [1.0, 1.483, 1.932, 2.546, 2.630, 3.897];
+
+		/**
+		 *  Harmonic ratio that produces a more pitched cymbal
+		 *  @private
+		 *  @static
+		 *  @type {Array}
+		 */
+		this.harmRatios = [1.0, 1.5, 2.025, 2.975, 4.0, 6.0];
+
+		/**
+		 *  A series of FMSynths
+		 *  @private
+		 *  @type {Array}
+		 */
+		this._oscillators = [];
+
+		/**
+		 *  Tone.ScaledSignals used for interpolating between harmonic
+		 *  and inharmonic ratios
+		 *  @private
+		 *  @type {Array}
+		 */
+		this._scaledSignals = [];
+
+		/**
+		 *  Tone.Multipliers for frequency multiplication of scaled signals
+		 *  @private
+		 *  @type {Array}
+		 */
+		this._freqMult  = [];
+
+		//make all the FMSynths, connect the frequency signals
+		for(var i = 0; i < 6; i++){
+			this._scaledSignals[i] = new Tone.Scale(this.inharmRatios[i], this.harmRatios[i]);
+			this._freqMult[i] = new Tone.Multiply();
+			this._oscillators[i] = new Tone.SimpleFM({
```

Comment:
Do you mean actually connect two oscillators into one signal

```
this.multiply = new Tone.Multiply().connect(this.strike);

this.osc1 = new Tone.Oscillator().connect(this.multiply, 0, 0);
this.osc2 = new Tone.Oscillator().connect(this.multiply, 0, 1);
```

I've tried a few configurations but I may just end up copying the SimpleFM to maintain the sound I have now but cut back on the unnecessary envelopes. 

With just two oscillators and a Tone.Multiply, how do you control ModulationIndex and Harmonicity?